src/options/App.vue

```diff
@@ -70,10 +70,8 @@
               </div>
               <div class="form-group row">
                 <label class="control-label col" for="gameUpdateInterval">
-                  Update your game list every
-                  {{ settings.gameUpdateInterval }} hour{{
-                    settings.gameUpdateInterval > 1 ? 's' : ''
-                  }}
+                  {{ $t('message.updateEvery') }}
+                  {{ settings.gameUpdateInterval }} {{ $t('message.hours') }}{{}}
```

Comment:
should be removed curly brace end of the line

```suggestion
                  {{ settings.gameUpdateInterval }} {{ $t('message.hours') }}
```